src/components/visualizations/chart/highcharts/comboConfiguration.ts

```diff
@@ -1,29 +1,47 @@
 // (C) 2007-2019 GoodData Corporation
-import get = require("lodash/get");
-import isEqual = require("lodash/isEqual");
 import { MAX_POINT_WIDTH } from "./commonConfiguration";
 import { LINE_WIDTH } from "./lineConfiguration";
 import { IChartConfig } from "../../../../interfaces/Config";
 import { VisualizationTypes } from "../../../../constants/visualizationTypes";
+import { isLineChart } from "../../utils/common";
 
 const { COLUMN, LINE } = VisualizationTypes;
+const DEFAULT_COMBO_CONFIG: IChartConfig = {
+    primaryChartType: COLUMN,
+    secondaryChartType: LINE,
+};
 
-export function getDefaultChartType(config?: IChartConfig) {
-    const primaryType = get(config, "primaryChartType", COLUMN);
-    const secondaryType = get(config, "secondaryChartType", LINE);
+export function getDefaultChartType(config: IChartConfig = DEFAULT_COMBO_CONFIG) {
```

Comment:
I think having the original `get(config, "primaryChartType", COLUMN);` was more safe. Default params are used only when the whole config would be missing. IChartConfig contains many other things, not just chart type config (such as colors, limits, etc.). So it would seem better not to assign DEFAULT_COMBO_CONFIG here because it will not be used when config will be given but will not contain primaryChartType and secondaryChartType.